data/transactions/logic/box_test.go

```diff
@@ -556,7 +556,7 @@ func TestBoxTotals(t *testing.T) {
 
 	ledger.NewApp(txn.Sender, 888, basics.AppParams{})
 	// The SENDER certainly has no boxes (but does exist)
-	logic.TestApp(t, `int 0; acct_params_get AcctTotalBoxes; pop; !`, ep)
+	logic.TestApp(t, `txn Sender; acct_params_get AcctTotalBoxes; pop; !`, ep)
```

Comment:
this does make for a better test app

Reply:
yeah, I was actually happy about some of the required changes. they read much better